cpplite/include/constants.dat

```diff
@@ -116,7 +116,7 @@ DAT(header_value_payload_format_nometadata, "application/json;odata=nometadata")
 DAT(header_value_payload_format_fullmetadata, "application/json;odata=fullmetadata")
 DAT(header_value_storage_blob_version, "2018-11-09")
 
-DAT(header_value_user_agent, "azure-storage-cpplite/0.3.0")
```

Comment:
There is no need to change this. We have taken this from the cpplite lib itself.

Reply:
No it should show Azure-storage-fuse and not cpplite. The logs are triaged based on Azure-storage-fuse. I understand it is not correct to change the Cpplite code base but the endproduct wins here. So the logs have to show Azure-storage-fuse. Leaving this as is